test/local/profile/updates.js

```diff
@@ -60,7 +61,7 @@ describe('profile updates', () => {
         assert.equal(mockLog.messages.length, 0)
         assert.equal(mockPush.notifyProfileUpdated.callCount, 2)
         var args = mockPush.notifyProfileUpdated.getCall(1).args
-        assert.deepEqual(args[0], Buffer.from(uid, 'hex'))
+        assert.deepEqual(args[0], uid)
```

Comment:
Although `deepEqual` works, we should probably change this to plain `equal` now that it's comparing strings.